open-craft/ecommerce
# OC-4529 Remove the "Verify your Identity" banner on ecommerce receipt page

Remove the "Verify your Identity" banner on ecommerce receipt page and replace it with a button to go to the LMS dashboard.

**Screenshot**
![receipt_page_no_verification_block](https://user-images.githubusercontent.com/218870/45165886-5da4bc00-b213-11e8-9879-f9ceadeb7e59.png)

**Testing instructions**
- Revert the changes in PR #3.
- Create a course on LMS.
- Add the course on the ecommerce portal under one of the paid options.
- Generate a '100% off' enrollment coupon for the course.
- Enroll a student using the enrollment coupon created in the previous step.
- Check that the 'Verify your Identity' banner does not show up and instead there is a button to navigate to the LMS dashboard.

**Reviewers**
